src/app/services/auth/auth.service.mock.ts

```diff
@@ -14,6 +18,15 @@ class MockAuthService {
   get isAuthenticated$() {
     return of(true);
   }
+
+  get authToken(): string {
+    return localStorage.getItem(LOCAL_STORAGE_AUTH_TOKEN_KEY) || '';
+  }
+
+  private setUserDataToLocalStorage(username: string, token: string) {
```

Comment:
If you make localStorage a provider with proper injection token, you will get rid of coupling with local or session storage, so setUserDataToStorage will be better option here

Reply:
Okay